pkgs/swiftgen/lib/src/ffigen_config.dart

```diff
@@ -0,0 +1,198 @@
+// Copyright (c) 2025, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:ffigen/ffigen.dart' as fg;
+import 'package:package_config/package_config.dart';
+
+class FfiGenConfig implements fg.Config {
```

Comment:
Hm, this makes adding fields to the FfiGen config a breaking change.

Maybe that's also not what we want either.

I guess if we only want to override some fields by providing these manually you want `extend` _and_ inherit all non-mentioned super constructor parameters. But Dart doesn't have a language feature for that. Adding parameters to the constructor is also a breaking change if you allow subclassing.

So maybe the first solution with copy-pasting is better? That would enable adding fields in FFIgen, and slowly catch up in Swiftgen without requiring major version revisions. 🙈  Sorry for the churn! Learning as we go here. 🙈 

I think we should give it some more thought how to structure all of this